pkg/artifactory/resource/security/resource_artifactory_scoped_token.go

```diff
@@ -305,7 +341,7 @@ func (r *ScopedTokenResource) Create(ctx context.Context, req resource.CreateReq
 
 	// Convert from Terraform data model into API data model
 	accessTokenPostBody := AccessTokenPostRequestAPIModel{
-		GrantType:             data.GrantType.ValueString(),
+		GrantType:             "client_credentials",
```

Comment:
This attribute has a default value of "client_credentials", no need to hardcode it here.